deps/rabbitmq_mqtt/src/rabbit_mqtt_packet.erl

```diff
@@ -7,19 +7,23 @@
 
 -module(rabbit_mqtt_packet).
 
--export([parse/2, initial_state/0]).
--export([serialise/2]).
-
 -include("rabbit_mqtt_packet.hrl").
 -include("rabbit_mqtt.hrl").
 
+-export([parse/2, initial_state/0, serialise/2]).
+-export_type([state/0]).
+
+-opaque state() :: 'none' | {more, any()}.
```

Comment:
Instead of `any()` we could be more specific here.
Also, `more` is not part of the state. The state is either `none` or a `fun`.

Reply:
fixed